src/components/Responsive/Visible/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import intersection from 'lodash/intersection';
+
+import Context from '../Context';
+
+const Visible = ({ children, breakpoints, ...passedProps }) => {
+  const getIsVisible = useCallback(
+    responsiveContext => !!intersection(breakpoints, responsiveContext).length,
```

Comment:
I think it's clearer if you just use `lodash/isEmpty`:

```suggestion
    responsiveContext => !isEmpty(intersection(breakpoints, responsiveContext)),
```